### ISSUE TITLE
Plans to release?

### ISSUE THREAD

**Issue by willemneal** [near/workspaces-js#127]
It seems it's been a long time since the last release: https://www.npmjs.com/package/near-workspaces

How are people using this project currently?

@ailisp

**Comment by ailisp** [comment]
We're working on significant breaking changes and plan to release a 2.0 soon. People are using 1.0 release and can still choose to use 1.0 for existing project

**Comment by willemneal** [comment]
Yeah but there are bug fixes currently being merged, so wondering if those will be released.

**Comment by ailisp** [comment]
@volovyk-s I don't have permission in npm and looks like you do. Could you release a new version from current `main` branch?

**Comment by volovyk-s** [comment]
released
